README.md

```diff
@@ -1,2 +1,31 @@
-# hw01
-Repository for homework 01
+# Jerry Shi
+![Me in DC over Spring Break! I had a great time there and gained at least five pounds from all the junk food I ate.](jerry.jpg)
+
+Intro
+=====
+
+Hi there! My name is Jerry Shi and I'm currently a Second Year Business Economics major at the University of Chicago. I'm writing this readme for the very first assignment of the class ["Computing for the Social Sciences"](https://cfss.uchicago.edu). In my spare time, some of my hobbies include
+
+ * crew
+ * yoga
+ * going to live concerts
+ * exploring downtown (which I *definitely* do not do enough!)
+ 
+A little more about me
+----------------------
+
+1. Hometown?
+
+I grew up all over the place! Born in Reno, then lived in SoCal (LA & San Diego) through middle school, then went to high school in Indiana.
+
+2. Favorite food?
```

Comment:
When writing enumerating list, you can write it in the code below, no need to spell out the numbered points.
```
1. point
    1. subpoint
    1. subpoint
```